extensions/ql-vscode/src/vscode-tests/factories/remote-queries/gh-api/variant-analysis-api-response.ts

```diff
@@ -2,14 +2,18 @@ import {
   VariantAnalysis as VariantAnalysisApiResponse,
   VariantAnalysisScannedRepository,
   VariantAnalysisSkippedRepositories,
+  VariantAnalysisStatus,
 } from '../../../../remote-queries/gh-api/variant-analysis';
 import {
   VariantAnalysisQueryLanguage
 } from '../../../../remote-queries/shared/variant-analysis';
+import { createMockScannedRepos } from './scanned-repositories';
+import { createMockSkippedRepos } from './skipped-repositories';
 
 export function createMockApiResponse(
-  scannedRepos: VariantAnalysisScannedRepository[],
-  skippedRepos: VariantAnalysisSkippedRepositories
+  status = 'in_progress',
```

Comment:
Should this have the `VariantAnalysisStatus` type instead of casting the string to `VariantAnalysisStatus` on line 29?